### ISSUE TITLE
does conda need a new environment for each installation？

### ISSUE THREAD

**Issue by neptuneyt** [conda/conda-build#4323]
Dear conda team,
Thanks for such amazing work.
I just confuse that shoud I create a new environment for each packages/toolkit installation for avoiding incompatible packages?

Looking someone reply, Thanks a lot.

**Comment by kenodegard** [comment]
Yes, anytime you are concerned with installing conflicting packages you should (probably) be creating a new environment.

I've personally deployed multiple versions of a package, each one siloed in its own environment for this very reason.

**Comment by cgr71ii** [comment]
I suppose that the question is related to conda build and not to conda itself.

When I create a new build, the dependencies of the recipe are not installed in the environment which I use to build the package, so why do we need to create a new environment for every package we build? In my case, I have a "conda_build" environment with `conda-build` and `conda-verify` installed which I use in order to build all my packages. Am I doing a bad practice?
